app/models/user/db_service.rb

```diff
@@ -200,7 +202,9 @@ def load_cartodb_functions(statement_timeout = nil, cdb_extension_target_version
           end
         end
 
-        upgrade_cartodb_postgres_extension(statement_timeout, cdb_extension_target_version)
+        if !load_cartodb_extension
```

Comment:
Don't we want to run it when it is true?

Should it be not - "if load_cartodb_extension" ?

Reply:
Wrote this initially as `skip_load_cartodb_extension` - must have missed this in the refactor.